src/transaction/log_recovery_redo_perf.hpp

```diff
@@ -121,9 +124,8 @@ class log_recovery_redo_perf_stat
 	  perf_stat_ss << "Log Recovery Redo statistics:" << std::endl;
 	  for (std::size_t perf_stat_idx = 0; perf_stat_idx < m_definition.get_value_count (); ++perf_stat_idx)
 	    {
-	      perf_stat_ss.imbue (std::locale (""));
 	      perf_stat_ss << '\t' << m_definition.get_value_name (perf_stat_idx)
-			   << " μs: " << perf_stat_results[perf_stat_idx] << std::endl;
+			   << ": " << perf_stat_results[perf_stat_idx] << std::endl;
```

Comment:
Wasn't it better when the time unit was added here?

Reply:
`us` was moved to label; because the entries are actually interspersed in the array: one is for the count, one for time